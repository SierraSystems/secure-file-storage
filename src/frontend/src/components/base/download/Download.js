import React from "react";
import { Files } from "../files/Files";
import "bootstrap/dist/css/bootstrap.css";

export const Download = () => {
  return (
    <div className="spacing">
      <p className="text head">Select a file to retrieve from S3:</p>
      <br />
      <Files />
    </div>
  );
};
