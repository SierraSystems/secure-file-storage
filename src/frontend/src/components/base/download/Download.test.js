import React from "react";
import { Download } from "./Download";
import testBasicSnapshot from "../../../TestHelper";

describe("Download", () => {
  test("Component renders as expected", () => {
    testBasicSnapshot(<Download />);
  });
});
