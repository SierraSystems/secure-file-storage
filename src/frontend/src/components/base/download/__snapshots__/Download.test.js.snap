// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Download Component renders as expected 1`] = `
<div
  className="spacing"
>
  <p
    className="text head"
  >
    Select a file to retrieve from S3:
  </p>
  <br />
  <div
    className="head"
  />
</div>
`;
